salesforce-backed-customer-insight-agent/sf_customer_360.py

```
@@ -0,0 +1,398 @@
+            changed_accts,
+            changed_opps,
+            lookback_hours=Settings.LOOKBACK_HOURS,
+            summary_text=summary_text,
+        )
+        res = post_slack(conn, Settings.SLACK_IDENTIFIER, Settings.DIGEST_CHANNEL_ID, text=txt, blocks=None)
+        ch = (res or {}).get("channel")
+        ts = (res or {}).get("ts") or (res or {}).get("timestamp")
+        meta = {"channel": ch, "ts": ts}
+        print("✅ Posted to Slack:", json.dumps(meta, indent=2))
+    else:
+        print("ℹ️ DIGEST_CHANNEL_ID not set; printing text message:")
```

Ruff (0.14.1)

String contains ambiguous `ℹ` (INFORMATION SOURCE). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)